buildspec.yaml

```diff
@@ -0,0 +1,50 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      docker: 20
```

Comment:
I was reading that for docker containers, you want to specify the docker version instead of the node version. Not sure if that's correct or not.

This file is essentially the same as the one in dmsp_backend_prototype, except for the node references.

Reply:
Ah ok cool. I didn't realize that.